lib/chai/core/assertions.js

```diff
@@ -172,10 +242,14 @@ module.exports = function (chai, _) {
   /**
    * ### .all
    *
-   * Sets the `all` flag (opposite of the `any` flag)
-   * later used by the `keys` assertion.
+   * Unsets the `any` flag, causing all `.keys` assertions that follow in the
+   * chain to revert to the default behavior of requiring that the target have
+   * all of the given keys, rather than only needing one or more of them.
```

Comment:
I think we could have an example here too. This is kind of an "advanced" feature and I think many newcomers will be confused if they see this.